### DISCUSSION TITLE
Suggestion - "Auto Run On" command plugin

### DISCUSSION THREAD

**Discussion by JAMEZ300HAHA** [runelite/runelite#15949] [Q&A] [answered]
I have this bug on RunLite where my run is glitching out causing me to enable running by constantly enabling run on and off.   A plugin that allows the user to automatically run when there is energy will be great.   For example at 0 energy the user will be walking but the character would run automatically when it recovers %1 energy when this type of plug in is on.   This way the auto run energy plugin would instantly toggle run "permanently on" when consuming energy restoring potions until the energy is at 0% again but also instantly running when any energy recovery happens.

**Comment by Enriath** [comment, marked as answer]
This could not be done without having the client send interaction to Jagex, which is botting.

The closes you can get is by using the ctrl walk option in the vanilla settings, and always holding control.
